ui/pages/ApplicationDetail.tsx

```diff
@@ -117,6 +144,83 @@ function ApplicationDetail({ className, name }: Props) {
         app={application}
         onAuthClick={() => setGithubAuthModalOpen(true)}
       />
+      <Modal
+        //confirm modal for app removal
+        open={removeAppModalOpen}
+        onClose={() => setRemoveAppModalOpen(false)}
+        title="Are You Sure?"
+        description={`You are about to remove ${application.name} from Weave GitOps`}
+      >
+        <Flex align column center wide>
+          {authSuccess ? (
+            <Flex align>
+              <Alert severity="success" message="Authentication Successful" />
+            </Flex>
+          ) : (
+            <>
+              <Flex center>
+                <Spacer padding="small">
```

Comment:
:+1: `<Spacer />` finally gets a little :sun_with_face:

Reply:
We love `<Spacer />` 💞